### DISCUSSION TITLE
Note about Timezone (Effective Only US [East/West] Feed)

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#188] [Show and tell]
Prevously, I Added NBC-CBS-FOX. 
[ET/PT]  in Last Part of Channel Name, Means Timezone.

●Why Adding Time Zone in Description?
It Will Affective Airing time Of Local News and Network's Morning Newscast(Ex. NBC's "TODAY").
Also Effective to Airtime of CW Prime Time Program.

Notice That Only Effective Actual Broadcast Stream. (Only East/West Stream)

**Comment by what-a** [comment]
First up, thank you for adding those channels! They are wonderful additions.

My thoughts regarding North America...
For national channels with different timezone streams (e.g. TBS, TNT, TCM) stating East or West in the name is indeed helpful.
However, for local channels (with or without network affiliation) the timezone should be omitted. Instead, always include the city and 2-letter state/province abbreviation. Doing so makes the timezone self evident (if not, one need only brush up a bit with geography).

**Reply by Central302** [reply]
Thanks, I'll Fix Next Time.
